### DISCUSSION TITLE
Miraculous fail of the solution

### DISCUSSION THREAD

**Discussion by junhyeok-yoon** [ANLAB-KAIST/KENSv3#215] [Project Q&A 2. Transport Layer (TCP)] [answered]
During test my unreliable data transfer, I checked 20000 random seeds and ONLY one seed failed. Whose random seed was 8024916 due to last handshake. 
And the solution failed either. When I checked packet log, I think the problem was mraculous 10 SYN packets loss. Thereby handshake time exceeds 1s as below
![image](https://github.com/ANLAB-KAIST/KENSv3/assets/114990411/fdc87029-d519-488f-9696-4bdd3376c882)

Should I ignore this seed?

**Comment by Pusnow** [comment, marked as answer]
You can ignore that `RANDOM_SEED`.
If all students fail on a certain `RANDOM_SEED`, I will exclude the seed for grading.
